src/calc.html

```diff
@@ -9,7 +9,7 @@
       }
       #contract {
         display: inline-block;
-        width: 400px;  /* TODO add alternate formatting for <400px */
+        width: 300px;  /* TODO add alternate formatting for <400px */
```

Comment:
delete the out of date comment, iphone is 320px wide so this is a perfectly capable mobile UI now.